e2e/core/settings-component.e2e.ts

```diff
@@ -0,0 +1,118 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { LoginPage } from '../pages/adf/loginPage';
+import { SettingsPage } from '../pages/adf/settingsPage';
+import { browser, element, protractor } from 'protractor';
+import TestConfig = require('../test.config');
+import { AcsUserModel } from '../models/ACS/acsUserModel';
+
+describe('Settings component', () => {
+
+    const loginPage = new LoginPage();
+    const settingsPage = new SettingsPage();
+
+    let adminUserModel = new AcsUserModel({
+        'id': TestConfig.adf.adminUser,
+        'password': TestConfig.adf.adminPassword
+    });
+
+    describe('Should be able to change Urls in the Settings', () => {
+        beforeEach( (done) => {
+            settingsPage.goToSettingsPage();
+            done();
+        });
+
+        it('[C245641] Should navigate User from Settings page to Login screen', () => {
+            settingsPage.clickBackButton();
+            loginPage.waitForElements();
+        });
+
+        it('[C291946] Should not save BPM Settings changes when User clicks Back button', () => {
+            settingsPage.setProvider(settingsPage.getBpmOption(), 'BPM');
+            settingsPage.setProcessServicesURL('http://adfdev.envalfresco1.com');
+            settingsPage.clickBackButton();
+            loginPage.waitForElements();
+            settingsPage.goToSettingsPage();
+            expect(settingsPage.getSelectedOptionText()).not.toEqual('BPM', 'The Settings changes are saved');
+            expect(settingsPage.getBpmHostUrl()).not.toEqual('http://adfdev.envalfresco1.com', 'The Settings changes are saved');
+
+        });
+
+        it('[C291947] Should not save ECM Settings changes when User clicks Back button', () => {
+            settingsPage.setProvider(settingsPage.getEcmOption(), 'ECM');
+            settingsPage.setContentServicesURL('http://adfdev.envalfresco1.com');
+            settingsPage.clickBackButton();
+            loginPage.waitForElements();
+            settingsPage.goToSettingsPage();
+            expect(settingsPage.getSelectedOptionText()).not.toEqual('ECM', 'The Settings changes are saved');
+            expect(settingsPage.getBpmHostUrl()).not.toEqual('http://adfdev.envalfresco1.com', 'The Settings changes are saved');
+
+        });
+
+        it('[C291948] Should save ALL Settings changes when User clicks Apply button', () => {
+            settingsPage.setProviderEcmBpm();
+            loginPage.waitForElements();
+            settingsPage.goToSettingsPage();
+            expect(settingsPage.getSelectedOptionText()).toEqual('ALL', 'The Settings changes are not saved');
+            expect(settingsPage.getBpmHostUrl()).toEqual('http://adfdev.envalfresco.com', 'The BPM Settings changes are not saved');
```

Comment:
We should use TestConfig.adf.url + TestConfig.adf.port instead of http://adfdev.envalfresco.com, in case someone wants to use a different env.